### DISCUSSION TITLE
xaxis modification

### DISCUSSION THREAD

**Discussion by paur94** [recharts/recharts#4072] [Q&A] [answered]
Hi there,

Is it possible to color the x-axis value if it corresponds to the current month and year, and ensure that this value is always displayed? Currently, it is sometimes not visible depending on the data. I would like the current month/year to be consistently shown and colored in green.

Here is my CodeSandbox link for reference:
[CodeSandbox Link](https://codesandbox.io/p/sandbox/rechart-axis-slzc5l?file=%2Fsrc%2FApp.js)

**Comment by ckifer** [comment, marked as answer]
Hey there, sure! At least kind of

Check this out [![Edit rechart-axis (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/rechart-axis-slzc5l?file=%2Fsrc%2FApp.js)

You can use the `tick` prop to support conditionally rendering based off of data in the XAxis.

You can use `interval` to adjust what the tick shows, (0 to show all) but you can't say "i want to make sure this specific tick is always showing" unless you set your own ticks using the `ticks` array. In that case you'll have to set all of them yourself
